gizmo.py

```diff
@@ -0,0 +1,97 @@
+# First draft for gizmo.py
+import numpy as np
+import pandas as pd
+
+def hello(name, country='Finland'):
+    print('Hello ' + name + ', how are things in ' + country + '?')
```

Comment:
[f-strings](https://realpython.com/python-f-strings/) are awesome for this type of work:

```python
print(f'(Hello {name}, how are things in {country}?')
```